_posts/2022-12-08-developer-survey-2022.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: Solita Developer Survey 2022 - summary of results
+author: aleksi.sitomaniemi
+
+excerpt: >
+ The bi-annual Solita Developer Survey was conducted again in 2022. Read more here to learn about the findings.
+tags:
+ - Software Development, Developer Survey
+
+---
+
+The bi-annual Solita Developer Survey was conducted in 2022 again, now with a participation of almost 300 people. Since 2020, the company headcount has grown with over _nnn_. The early 2022 acquisitions of Intellishore and Commentor from Denmark brought in a good one hundred people on board, the rest of the sum is new hires. The survey is targeted to all Solitans who are participating in development work regardless of their domain of expertise and title.
```

Comment:
Perhaps this section could mention the total head-count (and total n-of-devs) as well, besides the "increase since 2020".

Reply:
The information for 2020 does not seem to be easily available. I used wayback machine to check what was the rough number in 2020 for the company headcount (Over 1000) as compared to what we have today (Over 1500).  I reformatted this a bit to reflect both the total headcount and the increase in respondent number.